src/TrixiShallowWater.jl

```diff
@@ -1,11 +1,15 @@
 module TrixiShallowWater
 
 using Trixi: Trixi
```

Comment:
Yes, this was a conscious decision. The idea behind this was that in the package we will have to extend a lot of existing `Trixi.jl` functions. By including only `Trixi.jl` and qualifying the respective symbols that come from `Trixi.jl` it is directly clear in the code to what package a symbol belongs. 

To make this more clear that we do not want to import specific symbols from `Trixi.jl` it might be a good idea to change this to `import Trixi`?

Reply:
> Yes, this was a conscious decision.

Sounds good to me 👍



> To make this more clear that we do not want to import specific symbols from `Trixi.jl` it might be a good idea to change this to `import Trixi`?

I dunno. Experienced users will know what `using Trixi: Trixi` means, and that it is equivalent to `import Trixi`. Inexperienced users might benefit from `using Trixi: Trixi` in sense that they can see that it is _not_ just `using/`import`ing the module but something special is going on.

I have no strong feelings about it, but if you do not want to pull in all symbols anyways, I'd keep the `using Trixi: Trixi` style.